.github/workflows/dry-run.yml

```
@@ -2,7 +2,7 @@ name: Dry-run before publish
 
 on:
   pull_request:
-    branches: ["main"]
+    branches: [ "main" ]
```

yamllint (1.35.1)

[error] too many spaces inside brackets

(brackets)